src/models/marker/type.yaml

```
@@ -0,0 +1,8 @@
+type: object
+required:
+  - type
+description: "The type of the marker"
+properties:
+  type:
+    type: string
+    example: "credits" 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)